Wrappers/Python/cil/processors/RingRemover.py

```diff
@@ -145,7 +145,9 @@ def xRemoveStripesVertical(self,ima, decNum, wname, sigma):
         Corrected 2D sinogram data (Numpy Array)
         
         '''              
-                            
+
+        original_extent = [slice(s) for s in ima.shape]
```

Comment:
If this is just 2D I think it's more readable to just define it by hand rather than list comprehension.

Reply:
Thanks for reviewing Gemma :) I think I've updated this now